io.github.hakandundar34coding.system-monitoring-center.yml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.hakandundar34coding.system-monitoring-center
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: system-monitoring-center
+
+finish-args:
+  # For X11 shared memory access (higher performance for X11).
+  - --share=ipc
+  # For X11 access.
+  - --socket=x11
+  # For Wayland access.
+  - --socket=wayland
+  # For X11 fallback.
+  - --socket=fallback-x11
+  # For host OS commands access. Otherwise it gives this error: Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
+  - --socket=session-bus
```

Comment:
you need to determine which d-bus names you need.

Reply:
It gives the following error if ```- --socket=session-bus``` is removed:
```Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown```.
It is aded after using ```- --talk-name=org.freedesktop.Flatpak```. Otherwise the application is not run.

Reply:
Which is more secure if all of them work separately?

Reply:
The yml file is updated. It is built, installed and run without errors on my machine. But there are problems when it is built online. What is the problem?